File: problemas/iniciante/1186/1186.py
def main():
    O = input()
    soma = 0.0
    count = 0

    for i in range(12):
        for j in range(12):
            valor = float(input())

            if (j > 11 - i):
                soma += valor
                count += 1
    
    print(f"{soma:.1f}" if O == 'S' else f"{(soma / count):.1f}")

if __name__ == "__main__":
    main()

File: problemas/iniciante/1186/1186.js
const { endianness } = require("os");

process.stdin.setEncoding('utf-8');

let input = '';

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let entrada = input.trim().split('\n');
    const O = entrada.shift();
    
    let soma = 0.0;
    let count = 0;

    for (let i = 0; i < 12; ++i) {
        for (let j = 0; j < 12; ++j) {
            let valor = parseFloat(entrada.shift());

            if (j > 11 - i) {
                soma += valor;
                ++count;
            }
        }
    }

    console.log(`${(O === 'S' ? soma : soma / count).toFixed(1)}`);
});
